packages/core/ios/Fabric/RNLottieFabricConversions.h

```diff
@@ -0,0 +1,29 @@
+#ifdef __cplusplus
+#import <React/RCTConversions.h>
+
+inline NSArray<NSDictionary *> *convertColorFilters(std::vector<facebook::react::LottieAnimationViewColorFiltersStruct> colorFilterStructArr)
```

Comment:
Very similar to the old arch `LRNColorFilters` `RCTConvert` implementation

Reply:
Can we add some color filters in the example app to make sure this works properly?